Systems/Dialogue/BehaviorConversable.cs

```diff
@@ -210,7 +210,7 @@ public override void OnGameTick(float deltaTime)
                         sapi.Network.SendEntityPacket(player as IServerPlayer, entity.EntityId, CloseConvoPacketId);
                     } else
                     {
-                        (world.Api as ICoreClientAPI).Network.SendEntityPacket(entity.EntityId, CloseConvoPacketId);

```

Comment:
will be called in https://github.com/anegostudios/vssurvivalmod/blob/afc94069d30e0285185f5bf2dd798fe1b99fc2eb/Systems/Dialogue/BehaviorConversable.cs#L288